variables.tf

```diff
@@ -12,13 +12,17 @@ variable "records" {
   description = "List of names to create"
   type = list(object({
     name     = string
-    value    = string
+    value    = optional(string)
     type     = string
     ttl      = number
-    proxied  = bool
-    priority = number
+    proxied  = optional(bool)
```

Comment:
You have to address it also where you use this field. In case `proxied` is absent.

Reply:
Not sure I understand what you mean...

Reply:
Should you use `lookup` function here, for example?

https://github.com/Flaconi/terraform-cloudflare-records/blob/a778d4ce612984678377dd090100ab405ded4c6c/main.tf#L17

Reply:
solved